iroha/src/block.rs

```diff
@@ -147,11 +147,11 @@ impl SignedBlock {
                 Err(e) => eprintln!("Transaction validation failed: {}", e),
             }
         }
-        Ok(ValidBlock {
+        ValidBlock {
             header: self.header,
             signatures: self.signatures,
             transactions,
```

Comment:
Why would we have to store blocks without any transactions? (in the case when all transactions are invalid)

Reply:
That is a good question. The only thing that we have in the whitepaper right now is that the leader should not produce an empty block (block without transactions). But we should discuss the block validation in more detail. Maybe @humb1t can share some thoughts on this.